conference_schema.yml

```diff
@@ -0,0 +1,44 @@
+# Schema for conference data
+
+type:       map
+mapping:
+  "name":
+    type:      str
+    required:  True
+  "website":
+    type:      str
+    required:  True
+    pattern:   "https?://"
+  "location":
+    type:      str
+    required:  True
+  "status":
+    type:      str
```

Comment:
Tempted to make this an enum or whatever so that we limit the values that it can have. I guess we can do that later.